app/src/main/java/com/dcrandroid/activities/privacy/SetupPrivacy.kt

```diff
@@ -25,10 +29,27 @@ class SetupPrivacy : BaseActivity() {
         wallet_name.text = wallet.name
 
         btn_setup_mixer.setOnClickListener {
-            val intent = Intent(this, SetupMixerAccounts::class.java)
-            intent.putExtra(Constants.WALLET_ID, wallet.id)
-            finish()
-            startActivity(intent)
+            //Check for account name "imported" and subtract 1 from total account count
+            if (wallet.accounts.toString().contains("imported", ignoreCase = true)) {
+                val walletCount = wallet.accountsRaw.count - 1
+                if (walletCount > 1) {
```

Comment:
check if wallet is > 2, since we don't expect the default account to be used for privacy